CHANGELOG.md

```diff
@@ -112,6 +112,9 @@ Main (unreleased)
 
 - Fixed a bug where converting `YACE` cloudwatch config to river skipped converting static jobs. (@berler)
 
+- Fixed a bug due to which debug metrics originating from OpenTelemetry components
+  were not visible on the Agent't `metrics` endpoint. (@ptodev)
```

Comment:
You are correct - we have not released it yet, and I'll remove this line.

I added this line to the changelog because I thought #5045 must have been included in the 36.2 patch release. However, apparently it wasn't included.